crates/revm/src/lib.rs

```diff
@@ -1,9 +1,17 @@
 #![cfg_attr(not(feature = "std"), no_std)]
-#![warn(unreachable_pub)]
+#![warn(unused_crate_dependencies, unreachable_pub)]
 
 #[macro_use]
 extern crate alloc;
 
+// Actually used in benches and examples
+#[cfg(test)]
+mod unused {
+    use anyhow as _;
+    use criterion as _;
+    use ethers_contract as _;
+}
```

Comment:
Just curious is this for `rust-analyzer` or for `unused_crate_dependecies`?

Reply:
Actually I'll do that

Reply:
> Can also make the lint only enabled on `cfg_attr(test, .)`

It is fine as is, just curious